package.json

```diff
@@ -1,11 +1,16 @@
 {
   "name": "@google-cloud/debug-agent",
   "version": "2.1.3",
-  "author": "Google Inc.",
+  "author": {
+    "name": "Google Inc."
+  },
   "description": "Stackdriver Debug Agent for Node.js",
   "main": "./build/src/index.js",
   "types": "./build/types/index.d.ts",
-  "repository": "googlecloudplatform/cloud-debug-nodejs",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/googlecloudplatform/cloud-debug-nodejs.git"
```

Comment:
keep the original style. Is `gts` still making these changes? It seems we need to a new release to pick up https://github.com/google/ts-style/pull/55.

Reply:
gts 0.4.0 published. It will no longer pollute the `package.json`.